engine/access/ingestion/collections/syncer.go

```diff
@@ -28,20 +28,15 @@ const (
 	// if missing collections have been received during the initial collection catchup process.
 	DefaultCollectionCatchupDBPollInterval = 1 * time.Second
 
-	// DefaultMissingCollsRequestInterval is the interval at which the syncer checks missing collections
+	// DefaultMissingCollectionRequestInterval is the interval at which the syncer checks missing collections
 	// and re-requests them from the network if needed.
-	DefaultMissingCollsRequestInterval = 1 * time.Minute
+	DefaultMissingCollectionRequestInterval = 1 * time.Minute
 
-	// DefaultMissingCollsForBlockThreshold is the threshold number of blocks with missing collections
```

Comment:
removed this because we had redundant checks. if there are `MissingCollsForBlockThreshold` blocks with missing collections, then the last full block is at least `MissingCollsForAgeThreshold` blocks behind the last finalized. Updated to only check the difference between last finalized and last full